src/Directory.Build.targets

```diff
@@ -38,6 +41,10 @@
     <TestRunnerAdditionalArguments>$(XunitOptions)</TestRunnerAdditionalArguments>
     
     <PackageOutputPath Condition="'$(IsVisualStudioInsertionPackage)' == 'true'">$(DevDivPackagesDir)</PackageOutputPath>
+
+    <!-- Arcade sdk also carries an xunit.runner.json which sometimes overrides the one in this repo. Assign a value to the arcade properties XUnitDesktopSettingsFile and XUnitCoreSettingsFile to prevent the arcade version of the file being added. -->
```

Comment:
can you put some color on "sometimes"? Should I go yell at them?

Reply:
To be honest I didn't understand why it only happens on `Microsoft.Build.Engine.UnitTests.csproj` and not the rest. When I compare the logs to other projects, it all looks the same until the copy tasks does otherwise on that one project. So I just went with this fix.